### ISSUE TITLE
Dealing with atypical numbers of fingers

### ISSUE THREAD

**Issue by Manishearth** [immersive-web/webxr-hands-input#11]
Not everyone has the same number of fingers. What can we do to make things work well for these users? We should probably solicit input from affected users when tackling this problem.

Some concrete concerns:

 - We should enable applications to be written such that users may get a hand representation faithful to their own, if they so wish
 - It should be possible for applications using gestures to be able to support fallback gestures in cases where some fingers are missing

Some of this probably requires devices/backend APIs to expose more information (or configurability at the OS layer). Filing the issue now as a tracking issue so that we don't forget about it.

**Comment by kearwood** [comment]
There are other ways that a hand can vary, such as atypical finger lengths.
It may be desirable to normalize the hand model to avoid fingerprinting; however, this would require constraints to be defined (eg, optimize finger tip positions..  optimize finger angles regardless of length)

Generally, we may wish to strike a balance between user choice of physical representation, accessibility, and fingerprinting.

**Comment by rkochman** [comment]
Another use case: people with one or more partial fingers.

**Comment by Manishearth** [comment]
> Generally, we may wish to strike a balance between user choice of physical representation, accessibility, and fingerprinting.

Yeah, I imagine that a lot of this would be surfaced in the form of UA configurability.
